Description: Define PYTHON_PREFIX in make.tmpl.in for new ax-python.
 Newer ax-python may define @PYTHON3DIR@ which uses ${PYTHON_PREFIX}
 variable which is undefined, unless substitution for it exists in the
 makefile.
Author: Dimitri John Ledkov <dimitri.ledkov@canonical.com>

--- lvm2-2.03.11.orig/make.tmpl.in
+++ lvm2-2.03.11/make.tmpl.in
@@ -115,6 +115,7 @@ pythonprefix = $(DESTDIR)$(prefix)
 
 # N.B. No $(DESTDIR) prefix here.
 python2dir = @PYTHON2DIR@
+PYTHON_PREFIX = @PYTHON_PREFIX@
 python3dir = @PYTHON3DIR@
 
 USRLIB_RELPATH = @libdir@/
